form-submission-handler.js

```diff
@@ -19,13 +19,16 @@ function getFormData() {
   var data = {};
   fields.forEach(function(k){
     data[k] = elements[k].value;
+    var str = "";
     if(elements[k].type === "checkbox"){
-      data[k] = elements[k].checked;
+      str = str + elements[k].checked + ", ";
+      data[k] = str.slice(0, -2);
```

Comment:
might be useful to have an inline comment here explaining why we want `str.slice(0, -2)` ... 🤔